### ISSUE TITLE
Bug: perfs throws uncatchable error

### ISSUE THREAD

**Issue by pvampola** [elbywan/wretch#131]
Hello,

following code will throw `Uncaught (in promise) TypeError: Failed to fetch` as well as log 'Error'.

```
wretch('https://invalidXYZ.com')
    .get()
    .perfs(()=>{})
    .res()
    .catch(()=>console.log('Error'));
```

If I remove .perfs it will just log 'Error'.  I would expect just 'Error' in both cases.

It happens when url is not reachable, the same thing will happen with a working url and disconnected internet.

**Comment by elbywan** [comment]
Hey @pvampola,

> following code will throw Uncaught (in promise) TypeError: Failed to fetch as well as log 'Error'.

Thanks for reporting the issue! 🙇 
I just published 📦  `v1.7.10` which should include a fix.

**Comment by elbywan** [comment]
Closing the issue, feel free to reopen is needed.
